### PR TITLE
fixes and helps for development config

### PR DESCRIPTION
**Pull request by briskt**
### Added
- Added documentation details for development
- Added `--build` flag to docker compose to ensure the local images are built each time you start development.
- Use explicit tags on images pulled by docker compose 

### Feature branch checklist

- [x] Documentation (README, etc.)
- [ ] update version in package.json
- [ ] Run `make format` and `make depsupdate`

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -94,7 +94,7 @@ services:
 
   proxy:
     # https://github.com/silinternational/traefik-https-proxy
-    image: silintl/traefik-https-proxy
+    image: silintl/traefik-https-proxy:latest
```
Comment (hobbitronics) [inline docker-compose.yml:97]: Any reason your net setting an explicit version here?

File: docker-compose.yml [hunk 2]
```diff
@@ -250,7 +250,7 @@ services:
     command: bash -c "composer update && php restart.php"
 
   dynamo:
-    image: amazon/dynamodb-local
+    image: amazon/dynamodb-local:latest
```
Comment (hobbitronics) [inline docker-compose.yml:253]: Same question as above

**Review by hobbitronics — APPROVED** [review]
Thanks for doing this

File: docker-compose.yml (hunk 1 above)
Comment (briskt) [reply docker-compose.yml:97]: It's not available yet. [CVR-885](https://itse.youtrack.cloud/issue/CVR-885) push version tags for silintl/traefik-https-proxy

File: docker-compose.yml (hunk 2 above)
Comment (briskt) [reply docker-compose.yml:253]: They don't provide major or minor tags, only patch-level. I debated, but thought we probably wouldn't want to keep up with patch-level updates.

**Comment by briskt** [comment]
@hobbitronics should I wait on merging this until after you merge yours? (#189)

**Comment by hobbitronics** [comment]
> @hobbitronics should I wait on merging this until after you merge yours? (#189)

Since it should only affect developing I am fine either way

**Pull request merged**